### DISCUSSION TITLE
TypeError: null is not an object (evaluating 't.name.localeCompare')

### DISCUSSION THREAD

**Discussion by fbarnese** [code4recovery/12-step-meeting-list#1209] [General]
I've got an error in the meeting guide.  Need some assistance to getting this back online please.

**Comment by joshreisner** [comment]
oh no, sorry about that -- will fix

i think you may have a meeting without a name? what's your site? https://hacoaa.org/meetings/ looks ok

**Reply by joshreisner** [reply]
@fbarnese sorry about that! you should be fixed now if you do a hard refresh

i will look more closely at what happened, but i believe that you have a region with an empty name and it wasn't expecting that

**Reply by fbarnese** [reply]
I don't see an empty name in regions.

**Reply by joshreisner** [reply]
@fbarnese that is correct, this is a bug on our side - you didn't do anything wrong

there are two meetings `solid-solutions` and `sandy-survivors-2` that don't have regions, which is perfectly fine. TSML is outputting the wrong thing for these meetings:
```
"regions": [null]
```

when the key should just be undefined. that is causing this to be displayed:
<img width="342" alt="Screenshot 2023-09-01 at 11 34 39 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/1cb5111e-e579-40e2-a2ab-645954366f8c">

i'll create two tickets to fix it on the TSML and TSML UI sides. once the second ticket is done you will not see this gap anymore

**Reply by joshreisner** [reply]
(also i'll confirm the order is fixed)

**Reply by fbarnese** [reply]
You are the best Josh!  Thank you.

**Reply by joshreisner** [reply]
regions menu should be fixed now!

<img width="280" alt="Screenshot 2023-09-01 at 1 21 03 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/4c0a8fb6-0923-4ad5-83ef-c894ee499ac2">

**Comment by fbarnese** [comment]
It's the www.oc-aa.org<http://www.oc-aa.org> website.
________________________________

**Comment by fbarnese** [comment]
Many thanks Josh.

Sent from my iPhone

On Sep 1, 2023, at 1:22 PM, Josh Reisner ***@***.***> wrote:

﻿

regions menu should be fixed now!

[Screenshot 2023-09-01 at 1 21 03 PM]<https://user-images.githubusercontent.com/1551689/265120095-4c0a8fb6-0923-4ad5-83ef-c894ee499ac2.png>
